.github/workflows/node.js.yml

```diff
@@ -5,25 +5,24 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [ 18.x ]
```

Comment:
We don't really need a `strategy` anymore since we are testing only in the LTS version, anyway it's also ok to leave it as-is.

Not for this PR, but I think we should have a way to update this value to always match the one that we are using to build our frontend project.

Reply:
Removed section. Sounds reasonable to take a look at it later on.